GoogleDataTransport/GDTCCTLibrary/Protogen/nanopb/cct.nanopb.h

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2019 Google
+ * Copyright 2022 Google LLC
```

Comment:
Doesn't have to change, but if it does, it should be to 2023

Reply:
Thanks – I reverted the change. When running the proto generation script, it updates all the copyrights with copyright notice in the script. If it's alright with you, I might just let it update in the future so I need to revert all of related changes.